### ISSUE TITLE
Zapytanie odnośnie rate-limitu endpointów z centrum wiadomości

### ISSUE THREAD

**Issue by NotIdrizdi** [allegro/allegro-api#9991]
Witam, czy endpointy [GET /messaging/threads](https://developer.allegro.pl/documentation/#operation/listThreadsGET), [GET /messaging/threads/{threadId}/messages](https://developer.allegro.pl/documentation/#operation/listMessagesGET) posiadają jakikolwiek rate-limit?

**Comment by MaciejFrackowiak** [comment]
Nie, oprócz standardowego 9000 request/min nie ma tu żadnego dodatkowego limitu.
